ui/src/views/home/nfa-list/nfa-list.tsx

```diff
@@ -65,10 +65,13 @@ export const NFAList = () => {
           Next page
         </Button>
       </Flex>
-      <Flex css={{ gap: '$2' }}>
+      <div>

```

Comment:
seems like this `div` may be removed?

Reply:
this view will def change once we have the designs, but for now I need the div to display the items like this
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/52458215/224775013-135b499f-4b5e-4904-a1c9-00ece80203bf.png">